src/views/home/components/list/list.view.tsx

```diff
@@ -41,27 +41,27 @@ const List: FC<ListProps> = ({ list, onClose }) => {
         <Card className={classes.card}>
           <div className={classes.list}>
             {list.type === "chain"
-              ? list.items.slice(0, 20).map((element) => {
+              ? list.items.slice(0, 20).map((chain) => {
                   return (
                     <button
                       className={classes.button}
-                      key={element.name}
-                      onClick={() => list.onClick(element)}
+                      key={getChainName(chain)}
```

Comment:
I don't like this, we should use unique identifiers for the `key` and a label is not the best idea. If we use `chain.chainId` we can guarantee that this key is unique.

Reply:
Your right, I replaced `element.name` without realizing this should have been `element.key` in the first place.